app/Models/TeamCulture.php

```diff
@@ -40,12 +45,13 @@ class TeamCulture extends BaseModel {
         'gc_directory_url'
     ];
 
-    public function manager() {
+    public function manager()
+    {
         return $this->belongsTo(\App\Models\Manager::class);
     }
 
-    public function team_culture_translations() {
-        return $this->hasMany(\App\Models\TeamCultureTranslation::class);
+    public function team_culture_translations() //phpcs:ignore
+    {
+        return $this->hasMany(Astrotomic\Translatable\Translatable::class);
```

Comment:
Looks like this was replaced accidentally, should still be `\App\Models\TeamCultureTranslation::class`